tests/unit/Diff/Internal/AliasGroupListPatcherTest.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Tests\Diff\Internal;
+
+use Diff\DiffOp\Diff\Diff;
+use Diff\DiffOp\DiffOpAdd;
+use Diff\DiffOp\DiffOpChange;
+use Diff\DiffOp\DiffOpRemove;
+use PHPUnit_Framework_TestCase;
+use Wikibase\DataModel\Services\Diff\Internal\AliasGroupListPatcher;
+use Wikibase\DataModel\Term\AliasGroup;
+use Wikibase\DataModel\Term\AliasGroupList;
+
+/**
+ * @covers Wikibase\DataModel\Services\Diff\Internal\AliasGroupListPatcher
+ *
+ * @licence GNU GPL v2+
+ * @author Bene* < benestar.wikimedia@gmail.com >
+ */
+class AliasGroupListPatcherTest extends PHPUnit_Framework_TestCase {
+
+	public function providePatchAliasGroupList() {
+		return array(
+			'add aliases' => array(
+				new AliasGroupList(),
+				new Diff( array(
+					'en' => new Diff( array( new DiffOpAdd( 'foo' ), new DiffOpAdd( 'bar' ) ) )
+				) ),
+				new AliasGroupList( array(
+					new AliasGroup( 'en', array( 'foo', 'bar' ) )
+				) )
+			),
+			'add aliases (atomic)' => array(
+				new AliasGroupList(),
+				new Diff( array(
+					'en' => new DiffOpAdd( array( 'foo', 'bar' ) )
+				) ),
+				new AliasGroupList( array(
+					new AliasGroup( 'en', array( 'foo', 'bar' ) )
+				) )
+			),
+			'change aliases is no-op' => array(
+				new AliasGroupList( array(
+					new AliasGroup( 'en', array( 'foo', 'bar' ) )
+				) ),
+				new Diff( array(
+					'en' => new Diff( array( new DiffOpChange( 'bar', 'baz' ) ) )
```

Comment:
If I understand it correctly, that is basically what https://github.com/wmde/Diff/pull/65 is about. I just wanted to test the current behaviour but we should change this test if the other pull request gets merged.